src/core-scrabble.js

```diff
@@ -0,0 +1,38 @@
+const arrayToSum = (array) => {
+  let sum = 0
+  array.forEach((element) => (sum += element))
+  return sum
+}
+
+const arrayToNumbers = (array) => {
+  const datasheet = require('./datasheet.js').datasheet
+  const values = array.map((element) => datasheet[element])
+  return values
+}
+
+const stringToArray = (word) => {
+  const letters = []
+  word
+    .toLowerCase()
+    .split('')
+    .forEach((element) => {
+      letters.push(element)
+    })
+  return letters
+}
+
+const scrabble = (word) => {
+  if (word === null) {
+    return 0
+  } else if (typeof word === 'string') {
```

Comment:
Could this work by just making it an else statement? Because there's already the if statement above to check if it's null.